blueprint/core/blueprint_EcmascriptEngine.h

```diff
@@ -153,11 +154,12 @@ namespace blueprint
             static duk_ret_t callbackFinalizer (duk_context* ctx);
 
             juce::var::NativeFunction callback;
+            juce::Uuid                id;
         };
 
         //==============================================================================
         duk_context* ctx;
-        std::vector<std::unique_ptr<LambdaHelper>> lambdaReleasePool;
+        std::unordered_map<juce::Uuid, std::unique_ptr<LambdaHelper>> lambdaReleasePool;
```

Comment:
@nick-thompson, went for `unordered_map` in the end as it's easier to use with `unique_ptr`. I decided against a static counter in `LambdaHelper` as the key for the moment, could be inflexible in the future with multiple `EcmascriptEngine` instances? i.e. if you have multiple ReactApplicationRoot instances in your app etc.  

Had to bump juce to 5.4.2 to include the `std::hash` specialisation for `juce::Uuid`. Could use something else if necessary but I think this is OK ?

Reply:
Nice, I like it!

Yep, happy for the bump to 5.4.2. I want to bump to 6.x soon (#36) with CMake soon anyway. That should also make #22 quite easy. Much to do! :)